circuit_knitting/forging/entanglement_forging_ground_state_solver.py

```diff
@@ -317,14 +316,11 @@ def solve(
 
         # Create the EntanglementForgingResult from the results from the
         # results of eigenvalue minimization and other meta information
-        min_eigsolver_result = MinimumEigensolverResult()
-        min_eigsolver_result.eigenvalue = optimal_evaluation.eigenvalue
-        min_eigsolver_result.eigenstate = optimal_evaluation.eigenstate
-        result = EntanglementForgingResult.from_minimum_eigensolver_result(
-            min_eigsolver_result
-        )
-        result.energy_shift = self._energy_shift
+        result = EntanglementForgingResult()
+        result.eigenvalues = np.asarray([optimal_evaluation.eigenvalue])
+        result.eigenstates = [(optimal_evaluation.eigenstate, None)]
```

Comment:
Why are `eigenvalues` and `eigenstates` plural?  There is an `eigenvalue` (singular) property on `MinimumEigensolverResult` but I don't see `eigenstates` there or in `EntanglementForgingResult`.  And why is `eigenstates` a 2-tuple?  (Interesting to note that this didn't break any tests.)

Reply:
> Why are `eigenvalues` and `eigenstates` plural? There is an `eigenvalue` (singular) property on `MinimumEigensolverResult` but I don't see `eigenstates` there or in `EntanglementForgingResult`. And why is `eigenstates` a 2-tuple? (Interesting to note that this didn't break any tests.)

https://github.com/qiskit-community/qiskit-nature/blob/14ff6490c09af7091b90d1c38a99815a92b93a83/qiskit_nature/second_q/problems/eigenstate_result.py#L62-L63

The `groundstate` and `groundenergy` properties (the ones we really care about) are derived from these fields. `groundenergy` is the first element in the `eigenvalues` list, and `groundstate` is the first index of the first index of `eigenstates`. It didn't break any tests bc we've been using `EigenstateResult` as a base class the whole time.

Your comment brings up a larger question of whether we can break the dependency on this class. This would be in-line with all of the dependencies we've broken with Nature in the `EFGSSolver`, so I think the answer is yes. I've opened #275 to track this.

Reply:
Thanks. You clarified the source of my confusion, namely, that `EntanglementForgingResult` derives from `EigenstateResult` (which is in Qiskit Nature), not `MinimumEigensolverResult` (which is in Qiskit Terra).